### DISCUSSION TITLE
Can we get member activity for GitHub team?

### DISCUSSION THREAD

**Discussion by yousan** [community/community#22321] [New to GitHub]
<p>Hi.</p>
<p>We have a GitHub team at an organization.</p>
<p>We want to check my team members activity such as Issues, comments, commits, PRs…</p>
<p>Can we see the activity?</p>
<p>Thanks</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/yousan">@yousan</a>,</p>
<p>Thanks for being here! The audit log allows organization admins to quickly review the actions performed by members of your organization. It includes details such as who performed the action, what the action was, and when it was performed. For more details check out the <a href="https://help.github.com/articles/reviewing-the-audit-log-for-your-organization/" rel="nofollow noopener">documentation</a>.  I hope this helps!</p>
